extra-utils/bread-dog/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME="bread-dog"
+PKGDES="A clash proxy switcher like flying"
```

Comment:
Is "flying" a name of an application? But at any rate, you might want to be more descriptive. Rather than saying "like ..." just say how this package is.